packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -98,6 +103,81 @@ const spotEventPluginConfig = new ConfigureSpotEventPlugin(this, 'ConfigureSpotE
 });
 ```
 
+### Spot Event Plugin Fleet Subnet Placement
+
+We highly recommend creating dedicated subnets for the Spot Event Plugin Fleets in your farm as it is considered best practice and is especially important if you are using
+Deadline Secrets Management (see [Deadline Secrets Management Considerations](#using-dedicated-subnets-for-deadline-components)).
+
+The following example creates a dedicated subnet group for a `SpotEventPluginFleet`:
+```ts
+const vpc = new Vpc(this, 'Vpc', {
+  // ...
+  subnetConfiguration: [
+    // ...
+
+    // Provide a subnet configuration for the SpotEventPluginFleet subnet group
+    {
+      name: 'SpotEventPluginFleetSubnets',
+      subnetType: SubnetType.PRIVATE,
+      cidrMask: 20,
+    },
+
+    // ...
+  ],
+  // ...
+});
+
+// ...
+
+const fleet = new SpotEventPluginFleet(stack, 'SpotEventPluginFleet', {
+  // ...
+  vpc,
+
+  // Select the dedicated WorkerInstanceFleet subnets to put the worker nodes in
+  vpcSubnets: {
+    subnetGroupName: "SpotEventPluginFleetSubnets",
+  },
+
+  // ...
+});
+
+new ConfigureSpotEventPlugin(stack, 'ConfigureSpotEventPlugin', {
+  vpc,
+  spotFleets: [
+    fleet,
+  ],
+  // ...
+})
+```
+
+## Deadline Secrets Management Considerations
+
+### Using Dedicated Subnets for Deadline Components
+
+When using Deadline Secrets Management in RFDK, you will need to create [auto-registration rules](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html#automatically-registering-identities-and-adding-roles)
+for the various Deadline components that connect to your Repository or Render Queue. These auto-registration rules work like an IP-address-based allowlist that give machines with
```

Comment:
Two things:

1.  the way this is written, it sounds like the user must take action to create the identity registration settings.
2.  the Deadline documentation refers to the "auto-registration rules" as "identity registration settings" ([source](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html#identity-management-registration-settings-ref-label)). Can we use that terminology for consistency?

What do you think of changing from this:

>   When using Deadline Secrets Management in RFDK, **you will need to** create **auto-registration rules** for the various Deadline components that connect your Repository or Render Queue.

to:

>   When Secrets Management is enabled on the `Repository` construct, RFDK will handle configuring identity registration settings using the subnets of the RFDK constructs involved. Because a construct's resources can be assigned any IP address in the subnet range, RFDK must create rules that cover the entire subnet.

Reply:
Sounds good, updated